src/Template/Twig.php

```diff
@@ -0,0 +1,110 @@
+<?php
+/**
+ * Zend Framework (http://framework.zend.com/)
+ *
+ * @see       https://github.com/zendframework/zend-expressive for the canonical source repository
+ * @copyright Copyright (c) 2015 Zend Technologies USA Inc. (http://www.zend.com)
+ * @license   https://github.com/zendframework/zend-expressive/blob/master/LICENSE.md New BSD License
+ */
+
+namespace Zend\Expressive\Template;
+
+use Twig_Loader_Filesystem as TwigFilesystem;
+use Twig_Environment as TwigEnvironment;
+
+/**
+ * Template implementation bridging league/plates
+ */
+class Twig implements TemplateInterface
+{
+    /**
+     * @var TwigFilesystem
+     */
+    protected $twigLoader;
+
+    /**
+     * @var TwigEnvironment
+     */
+    protected $template;
+
+    /**
+     *  Constructor
+     *
+     * @param TwigEnvironment $template
+     */
+    public function __construct(TwigEnvironment $template = null)
+    {
+        if (null === $template) {
+            $template = $this->createTemplate($this->getDefaultLoader());
```

Comment:
Nice! I didn't like putting the `getDefaultLoader()` call in `createTemplate()`; this is much better!